packages/react/src/components/UIShell/SideNavLink.js

```diff
@@ -16,12 +16,15 @@ import SideNavLinkText from './SideNavLinkText';
 
 const { prefix } = settings;
 
+
 const SideNavLink = ({
   className: customClassName,
   children,
   renderIcon: IconElement,
   isActive,
   large,
+  // eslint-disable-next-line no-unused-vars
```

Comment:
Thank you for the update; BTW I thought https://twitter.com/dan_abramov/status/838576620235550720 is applied to our ESLint config. Did you see it's not the case...?

Reply:
Before I pushed the first commit, I didn't disable any eslint rules. I ran "yarn ci-check" on my local machine and then faced the error "no-unused-vars."

Reply:
OK, fair enough - Thank you for clarifying!